.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## Description
+
+Please include a summary of the change and be sure you follow the contributions rules we do provide [here](./CONTRIBUTIONS.md)
+
+Task ID: [OS-?](https://aragonassociation.atlassian.net/browse/OS-?)
+
+## Type of change
+
+See the framework lifecylce in `packages/contracts/docs/framework-lifecycle` to decide what kind of change this pull request is.
+
+<!--- Please delete options that are not relevant. -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Checklist:
+
+- [ ] I have selected the correct base branch.
+- [ ] I have performed a self-review of my own code.
+- [ ] I have commented my code, particularly in hard-to-understand areas.
+- [ ] My changes generate no new warnings.
+- [ ] Any dependent changes have been merged and published in downstream modules.
+- [ ] I ran all tests with success and extended them if necessary.
+- [ ] I have updated the `CHANGELOG.md` file in the root folder.
+- [ ] I have updated the `DEPLOYMENT_CHECKLIST` file in the root folder.
+- [ ] I have updated the `UPDATE_CHECKLIST` file in the root folder.
```

Comment:
None of those files are in the root folder, I think we could let the changelog and maybe add a Readme, wdyt?

```suggestion
- [ ] I have updated the `CHANGELOG.md` file.
- [ ] I have updated the `README` in the root folder.
```

Reply:
Will create tasks for this